templates/docs/examples/patterns/tiered-list/full-width-with-description.html

```diff
@@ -6,7 +6,7 @@
 {% block standalone_css %}patterns_all{% endblock %}
 
 {% block content %}
-{%- call(slot) tiered_list(is_description_full_width_on_desktop=true, is_list_full_width_on_tablet=true) -%}
+{%- call(slot) tiered_list(variant=4) -%}
   {%- if slot == 'title' -%}
```

Comment:
My readability concern is that the user can't tell anything about what `variant=4` means just by looking at this macro call